Tasks/AzureRmWebAppDeployment/dependencies.json

```diff
@@ -10,6 +10,24 @@
               "MSDeploy3.6/Microsoft.Web.Deployment.Tracing.dll",
               "MSDeploy3.6/msdeploy.exe"
             ]
+        },
+        {
+            "archiveName": "7zip.zip",
+            "url": "https://vstsagenttools.blob.core.windows.net/tools/7zip/1/7zip.zip",
+            "dest": "./",
+            "files": [
+              "7zip/7z.exe",
```

Comment:
@RoopeshNair can you confirm if this confirms our OSS requirements? We have the dependency in a explicit folder.